misbehave.html

```diff
@@ -55,6 +55,17 @@
         behaviorDefinition[key] = thisPrototype[key];
       }
 
+      behaviorDefinition.injectFrom = function(base, target) {
+        var baseElement = document.createElement(base);
```

Comment:
What if `base` isn't an element that's being mixed in to the ext component?  I suppose this function could care and enforce that, or it could work for any `base` element type.  What do you think?